### PR TITLE
Div API fixes

### PR DESCRIPTION
**Pull request by cableman**
* Set agency from auth
* Updated API required fields
* Remove auth not implemented note
* Event subscriber renamed

### CODE REVIEW

**Comment by cableman** [comment]
The last check is about the 120 char line limit in migration files..

File: src/Entity/Material.php [hunk 1]
```diff
@@ -5,16 +5,33 @@
 use ApiPlatform\Core\Annotation\ApiProperty;
 use ApiPlatform\Core\Annotation\ApiResource;
 use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Serializer\Annotation\Groups;
 
 /**
  * @ApiResource(
+ *     normalizationContext={
+ *          "groups"={"material:read"},
```
Comment (tuj) [inline src/Entity/Material.php:13]: Alignment

**Review by turegjorup — APPROVED** [review]

File: src/EventSubscriber/MaterialPostWriteSubscriber.php [hunk 2]
```diff
@@ -14,25 +18,45 @@
...
-            KernelEvents::VIEW => ['uploadImage', EventPriorities::POST_WRITE],
+            KernelEvents::VIEW => [
+                'materialPostWrite', EventPriorities::POST_WRITE,
+            ],
         ];
     }
 
-    public function uploadImage(ViewEvent $event)
+    /**
+     * Send event into the queue system to trigger upload an indexing.
```
Comment (tuj) [inline src/EventSubscriber/MaterialPostWriteSubscriber.php:54]: an => and

**Review by tuj — APPROVED** [review]
A few minor. Approved.

**Pull request merged**